packages/bits/src/lib/sorter/sorter.component.ts

```diff
@@ -193,10 +207,18 @@ export class SorterComponent implements OnChanges, OnInit, OnDestroy, AfterViewI
     }
 
     public ngOnDestroy() {
+        this.menuKeyControlListeners.forEach(listener => listener());
         this.onDestroy$.next();
         this.onDestroy$.complete();
     }
 
+    public toggleSorterMenu(): void {
+        if (!this.overlay.showing) {
+            this.sorterKeyControlService.announceDropdown(true);
+        }
+        this.overlay.toggle();
```

Comment:
Shouldn't we also announce when it's toggled closed?

Something like this:

```suggestion
        this.overlay.toggle();
        this.sorterKeyControlService.announceDropdown(this.overlay.showing);
```